rfc/0016-specification-repository.md

```diff
@@ -0,0 +1,54 @@
+# Summary
+
+This RFC amends the RFC process with the creation of a specifications repository, which holds the body of some policy defined through RFCs here.
+
+This RFC does not seek to change the way policy and documentation is reviewed and accepted, only how it is amended and referenced.
+
+# Motivation
+
+Currently, policy on team structure, voting, and other critical aspects of governance are defined only through RFCs.
+This is an issue when a major part of that RFC needs to be redefined--the entire RFC template is excluded for these changes! This is fine for clarifications, but in large-scale changes that require lots of discussion, the current process works against the point of defining these documents through RFCs in the first place.
```

Comment:
This shouldn't be an issue. If there needs to be made a small tweak, the RFC can be amended. Otherwise a new RFC can be made replacing the old RFC.